src/mixxxmainwindow.cpp

```diff
@@ -1095,6 +1095,16 @@ void MixxxMainWindow::rebootMixxxView() {
     bool wasFullScreen = isFullScreen();
     slotViewFullScreen(false);
 
+    // Reload default styles that can be overridden by skins
+    QFile file(":/skins/default.qss");
+    if (file.open(QIODevice::ReadOnly)) {
+        QByteArray fileBytes = file.readAll();
+        QString style = QString::fromLocal8Bit(fileBytes);
```

Comment:
I don't think that fromLocal8Bit is correct here, because default.qss has always the same encoding on all our targets. 
Is it Latin1 or Utf8?

Reply:
ParserM3u uses Utf8 :shrug: 
I moved the redundant parsing to an inline function. We could as well move it to a member function (incl. setStyleSheet()).

Reply:
> an inline function

that was pointless, moved it to a member function.